rooms.js

```diff
@@ -1037,6 +1037,9 @@ let BattleRoom = (() => {
 		logData.endType = this.battle.endType;
 		if (!p1rating) logData.ladderError = true;
 		const date = new Date();
+		logData.timestamp = date;
```

Comment:
Um, JSON doesn't support storing dates. It will `toJSON` the date, which I believe makes it a string. Is that what you want, or do you want a numeric timestamp?